ml_git/commands/custom_options.py

```diff
@@ -114,3 +116,15 @@ def check_multiple(ctx, param, value):
     elif len(value) > 1:
         raise click.BadParameter(output_messages['ERROR_OPTION_WITH_MULTIPLE_VALUES'].format(param))
     return value[0]
+
+
+def check_valid_storage_choise(ctx, param, value):
```

Comment:
```suggestion
def check_valid_storage_choice(ctx, param, value):
```
In some places in the wizard we are using the term `choise`, which is an obsolete/archaic spelling of choice, let's replace it with `choice`.

Reply:
The value was replaced at [7ff0862](https://github.com/HPInc/ml-git/pull/228/commits/7ff086215b0254e686de1328abf5f0a1b3a42038)